spec/factories/courses.rb

```diff
@@ -130,6 +131,11 @@
         course.site_statuses << site_status
       end
 
+      course.subjects = []
```

Comment:
don't you need to check `evaluator.subjects.any?` first before over writing it

Reply:
There's no need since we don't currently set any default subjects.